content/en/variables/page.md

```diff
@@ -60,6 +63,9 @@ See [`.Scratch`](/functions/scratch/) for page-scoped, writable variables.
 .Hugo
 : see [Hugo Variables](/variables/hugo/).
 
```

Comment:
Images and Videos are undocumented for a reason. The design wasn't really thought through and we will replace it with "remote resources" or some thing, so we get "one thing".

Reply:
Understandable, I dropped these commits. I'm going to go over and check what remaining variables I can add that have documentation.